app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -219,24 +215,24 @@ class CtaViewModelTest {
     }
 
     @Test
-    fun whenRefreshCtaOnNewTabThenReturnDaxIntroCta() = runBlockingTest {
-        setConceptTestVariant()
+    fun whenRefreshCtaOnNewTabAndConceptTestFeatureActiveThenReturnDaxIntroCta() = runBlockingTest {
```

Comment:
Hey @subsymbolic, https://github.com/duckduckgo/Android/pull/673 gave you some conflicts here. Happy to help you in solving this later today.

I will still put some context here:
* we renamed/mapped `isNewTab` to `isBrowserShowing` to show a better intention inside our `ctaViewModel.refreshCta`.
`isNewTab = false` equals `isBrowserShowing = true`
`isNewTab = true` equals `isBrowserShowing = false`
* testcases names have been affected by the change too:
`whenRefreshCtaOnNewTab` -> `whenRefreshCtaOnHomeTab`
`whenRefreshCtaOnExistingTab` -> `whenRefreshCtaWhileBrowsing`

Reply:
Thanks!